HIP/hip-868.md

```diff
@@ -0,0 +1,155 @@
+destroyed as per prior behavior.
+
+However, since Hedera places "ownership" of token balances with the account
+holding the balance we will need to extend the behavior of the sweep mode to
+also transfer all HTS tokens to the beneficiary. Notably, for a successful
+transaction beneficiary accounts must support the receiving of HTS tokens via
+association configuration. This is consistent with current self-destruct
+behavior and consistent with the spirit of the Ethereum change in that code and
+the account are no longer deleted.
+
+When the self-destruct is invoked on a contract within a transaction the
+behavior of Hedera will remain as it currently is in all respects.
+
+## User stories
+
+* As a smart contract developer I want to use the "sweep" semantics of EIP-6780
+  in my contracts to sweep out value (HBAR and HTS token balance) while not
+  destroying the contract.
+* As a smart contract developer I want to be able to continue to use the
+  transient contract pattern and have it behave the same way prior to this HIP.
+* As a HAPI user I want to be able to control my contract account in the same
+  ways I did prior to this HIP.
+
+## Specification
+
+There are no changes needed to the JSON-RPC relay.
+
+### EVM Implementation
+
+The SelfDestructOperation will be re-implemented with hedera semantics. An
+existing hedera version of self-destruct exists that may be manipulated in a way
+similar to how the Besu EVM's implementation was altered.
+
+The existing transient self-destruct behaviors will be retained, and the new
+sweep mode changes will be added. All hBar and HTS tokens will be swept as part
+of the self-destruct to the beneficiary address, what will change is that the
+contract will not be marked for removal.
+
+### Hedera Services
+
+Hedera services may need to be changed to ensure that the transaction processing
+does not leave a "shadow" account number for transient self-destructs. Such
+transient self-destructs will also need to be tested to ensure all HTS tokens
+are properly swept during the sweep as well as the transient contract modes.
+
+## Backwards Compatibility
+
+This change could break contracts that depend on selfdestruct destroying old
+contracts. Relying on such behaviors has been discouraged for years and formally
+deprecated for at least a year on Ethereum Mainnet, so it is expected that such
+use cases have never found their way onto Hedera's Mainnet.
+
+In the event such a pattern is discovered, the ability to use HAPI calls to
+delete the contract can be used as a shim to restore proper functionality
+between transactions.
+
+## Security Implications
+
+The substantial changes of this HIP are the removal of some functionality in
+some circumstances. There are no known security impacts of this removal and
+furthermore there are no known security mitigations that require the presence of
+the removed functionality. It is expected the security implications will be
```

Comment:
"minimal to none" -> why not just "none" then?

(Although I see that [EIP-6780](https://eips.ethereum.org/EIPS/eip-6780#security-considerations) says something more specific anyway - does it need to be repeated here?

<img width="1140" alt="image" src="https://github.com/hashgraph/hedera-improvement-proposal/assets/117694041/3d3871a1-bfe0-4405-ac67-daf9e97ffa98">


)